pkg/api/types.go

```diff
@@ -123,8 +123,8 @@ type OrchestratorProfile struct {
 // KubernetesConfig contains the Kubernetes config structure, containing
 // Kubernetes specific configuration
 type KubernetesConfig struct {
-	KubernetesHyperkubeSpec string `json:"kubernetesHyperkubeSpec,omitempty"`
-	KubectlVersion          string `json:"kubectlVersion,omitempty"`
+	KubernetesImageBase string `json:"kubernetesImageBase,omitempty"`
+	KubectlVersion      string `json:"kubectlVersion,omitempty"`
```

Comment:
We only use the version to download `kubectl` though. If we just extract `kubectl` on the node from the `hyperkube` image, we don't need to construct the kubectl url at all, or download it at all.

It also has the benefit of meaning that if you use a custom hyperkube build with... say a new Disk type... you will automatically get the correct `kubectl`. Otherwise, you'd wind up with the wrong `kubectl` that would prevent you from testing the disk change properly... etc...